servers/rendering/storage/render_scene_buffers.cpp

```diff
@@ -61,11 +61,15 @@ void RenderSceneBuffersConfiguration::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_fsr_sharpness"), &RenderSceneBuffersConfiguration::get_fsr_sharpness);
 	ClassDB::bind_method(D_METHOD("set_fsr_sharpness", "fsr_sharpness"), &RenderSceneBuffersConfiguration::set_fsr_sharpness);
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "fsr_sharpness"), "set_fsr_sharpness", "get_fsr_sharpness");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "fsr_sharpness"), "set_fsr_sharpness", "get_fsr_sharpness");
```

Comment:
This is an interesting error, but I wouldn't have realized it it was part of this PR if I didn't inspect it. Perhaps it would be best to have this in its own PR so it can be reviewed independently what it was doing?

Reply:
Might've slid in since I'm using these changes on my own custom godot build, just seemed to me while changing the files that someone might accidentally left those as `BOOL`. I'll remove those and maybe make a new PR just for that, although I have no clue what they really do.

Reply:
I think it sounds like a bug as well, the parameter is clearly meant to be a float. But I'd definitely make it part of a separate PR to track it down better and not have to associate that fix to this feature upgrade in case we want to cherry-pick or do regression testing.

Reply:
The property hint type is just a suggestion for the inspector and class reference, but it should be fixed in a separate PR indeed.